src/index.js

```diff
@@ -63,23 +75,39 @@ export const mapMultiRowFields = normalizeNamespace((
 
   return Object.keys(pathsObject).reduce((entries, key) => {
     const path = pathsObject[key];
-
     // eslint-disable-next-line no-param-reassign
     entries[key] = {
       get() {
+        // eslint-disable-next-line no-unused-vars
+        let param = null;
```

Comment:
Why do we need unused vars 🤔

Reply:
It's being used below. I think the wrong thing is the //eslint-disable